support/jaxrs/pom.xml

```diff
@@ -45,8 +45,8 @@
         </dependency>
 
         <dependency>
-            <groupId>javax.ws.rs</groupId>
-            <artifactId>javax.ws.rs-api</artifactId>
+            <groupId>jakarta.ws.rs</groupId>
+            <artifactId>jakarta.ws.rs-api</artifactId>
```

Comment:
set the scope to `provided` and we should be good

Reply:
Done.